<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTrendadisplay</class>
 <widget class="QWidget" name="QTrendadisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_rulerScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>标尺刻度</string>
   </property>
   <widget class="QSpinBox" name="spinBox_verticalMain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_horizMain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_horizSub">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_verticalSub">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_verticalSub">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>31</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直次标尺：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_horizSub">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>64</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水平次标尺：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_vertical">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>显示垂直主标尺刻度</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_horiz">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>显示水平标尺刻度</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直主标尺：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>水平主标尺：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_otherColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>颜色设置</string>
   </property>
   <widget class="QGroupBox" name="groupBox_grade">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>网格</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直网格颜色：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_verGradeColor">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>水平网格颜色：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_horGradeColor">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_rulerColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>标尺色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rulerColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tableColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>103</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图表区色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_frameColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tableColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>边框色：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>数据样本</string>
   </property>
   <widget class="QSpinBox" name="spinBox_dataSampleNum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>18</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="suffix">
     <string notr="true"/>
    </property>
    <property name="prefix">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_dataSampleNum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>数据样本数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rollSampleNum">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>17</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_rollSampleNum">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>18</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>滚动样本数：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>文本</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>18</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文本颜色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>16</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rollSampleNum_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>18</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
